src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppInsights.java

```diff
@@ -16,6 +19,10 @@
 
     static final String LETTER_NOT_PRINTED = "LetterNotPrinted";
 
+    static final String LETTER_PRINT_REPORT = "LetterPrintReport";
```

Comment:
Could we use an enum for these values?

Reply:
Enum is not really valuable in this case. We still need string as a telemetry name and there is nothing to compare to or keep any control of it functionally.

Reply:
It would be nice if App insights took an object as an event name and called toString on it, then one could pass either a string or an enum to it. Nevermind though.

Reply:
I'm not sure LetterPrintReport makes sense as an event name because it doesn't have a verb; perhaps LetterPrintReportProcessed or LetterReportProcessed or XeroxReportProcessed?